assets/i18n/hi.json

```diff
@@ -0,0 +1,276 @@
+  "I18N_INTERACTIONS_LOGIC_PROOF_INCORRECT_ANSWERS": "ग़लत: <[answer]>",
+  "I18N_INTERACTIONS_LOGIC_PROOF_POSSIBLE_LINES": "संभव लाइनों",
+  "I18N_INTERACTIONS_LOGIC_PROOF_QUESTION_STR_ASSUMPTIONS": "यह सोचते हुए  <[assumptions]>; साबित करे <[target]>.",
+  "I18N_INTERACTIONS_LOGIC_PROOF_QUESTION_STR_NO_ASSUMPTION": "साबित करे  <[target]>.",
+  "I18N_INTERACTIONS_LOGIC_PROOF_SUBMIT": "जमा करें",
+  "I18N_INTERACTIONS_MUSIC_CLEAR": "मिटाये",
+  "I18N_INTERACTIONS_MUSIC_PLAY": "चलाएँ",
+  "I18N_INTERACTIONS_MUSIC_PLAY_SEQUENCE": "लक्ष्य अनुक्रम चलाएँ",
+  "I18N_INTERACTIONS_MUSIC_SUBMIT": "जमा करें",
+  "I18N_INTERACTIONS_SET_INPUT_ADD_ITEM": "जोडें",
+  "I18N_INTERACTIONS_SET_INPUT_DUPLICATES_ERROR": "ओह, यह लग रहा है की आपके सेट में कुछ सामान आइटम है !",
+  "I18N_INTERACTIONS_SET_INPUT_EMPTY_SET": "(प्रति लाइन एक आइटम जोड़े। )",
+  "I18N_INTERACTIONS_SET_INPUT_NO_ANSWER": "कोई जवाब नहीं दिया गया है। ",
+  "I18N_INTERACTIONS_SET_INPUT_SUBMIT": "जमा करें",
+  "I18N_LANGUAGE_FOOTER_VIEW_IN": "Oppia को देखे:",
+  "I18N_LIBRARY_ALL_CATEGORIES": "सभी वर्ग",
+  "I18N_LIBRARY_ALL_CATEGORIES_SELECTED": "सभी वर्गो में चयनित",
+  "I18N_LIBRARY_ALL_LANGUAGES": "सारी भाषाएँ",
+  "I18N_LIBRARY_ALL_LANGUAGES_SELECTED": "सारी भाषाएँ चयनित",
+  "I18N_LIBRARY_CATEGORIES_ALGORITHMS": "एल्गोरिदम",
+  "I18N_LIBRARY_CATEGORIES_ARCHITECTURE": "आर्किटेक्चर",
+  "I18N_LIBRARY_CATEGORIES_ART": "कला",
+  "I18N_LIBRARY_CATEGORIES_BIOLOGY": "जीवविज्ञान",
+  "I18N_LIBRARY_CATEGORIES_BUSINESS": "व्यापार",
+  "I18N_LIBRARY_CATEGORIES_CHEMISTRY": "रसायन विज्ञान",
+  "I18N_LIBRARY_CATEGORIES_CODING": "कोडिंग",
+  "I18N_LIBRARY_CATEGORIES_COMPUTING": "कम्प्यूटिंग",
+  "I18N_LIBRARY_CATEGORIES_ECONOMICS": "अर्थशास्त्र",
+  "I18N_LIBRARY_CATEGORIES_EDUCATION": "शिक्षा",
+  "I18N_LIBRARY_CATEGORIES_ENGINEERING": "अभियांत्रिकी",
+  "I18N_LIBRARY_CATEGORIES_ENGLISH": "अंग्रेज़ी",
+  "I18N_LIBRARY_CATEGORIES_ENVIRONMENT": "वातावरण",
+  "I18N_LIBRARY_CATEGORIES_GEOGRAPHY": "भूगोल",
+  "I18N_LIBRARY_CATEGORIES_GOVERNMENT": "सरकार",
+  "I18N_LIBRARY_CATEGORIES_HISTORY": "इतिहास",
+  "I18N_LIBRARY_CATEGORIES_HOBBIES": "शौक",
+  "I18N_LIBRARY_CATEGORIES_INTERACTIVE_FICTION": "इंटरएक्टिव कथा",
+  "I18N_LIBRARY_CATEGORIES_LANGUAGES": "भाषाएँ",
+  "I18N_LIBRARY_CATEGORIES_LAW": "कानून",
+  "I18N_LIBRARY_CATEGORIES_LIFE_SKILLS": "जीवन कौशल",
+  "I18N_LIBRARY_CATEGORIES_MATHEMATICS": "अंक शास्त्र",
+  "I18N_LIBRARY_CATEGORIES_MATHS": "गणित",
+  "I18N_LIBRARY_CATEGORIES_MEDICINE": "दवा",
+  "I18N_LIBRARY_CATEGORIES_MUSIC": "संगीत",
+  "I18N_LIBRARY_CATEGORIES_PHILOSOPHY": "दर्शन",
+  "I18N_LIBRARY_CATEGORIES_PHYSICS": "भौतिक विज्ञान",
+  "I18N_LIBRARY_CATEGORIES_PROGRAMMING": "प्रोग्रामिंग",
+  "I18N_LIBRARY_CATEGORIES_PSYCHOLOGY": "मनोविज्ञान",
+  "I18N_LIBRARY_CATEGORIES_PUZZLES": "पहेलि",
+  "I18N_LIBRARY_CATEGORIES_READING": "पढ़ना",
+  "I18N_LIBRARY_CATEGORIES_RELIGION": "धर्म",
+  "I18N_LIBRARY_CATEGORIES_SPORT": "खेल",
+  "I18N_LIBRARY_CATEGORIES_STATISTICS": "सांख्यिकी",
```

Comment:
How about "अंकशास्त्र" rather than "सांख्यिकी" ?
Personally I think "अंकशास्त्र" is better understood. Choice is yours, both seems fine.